packages/blockchain-wallet-v4-frontend/src/data/components/bsvTransactions/sagas.js

```diff
@@ -1,8 +1,9 @@
 import { select, put } from 'redux-saga/effects'
 import { equals, path, prop } from 'ramda'
-import { actions, selectors } from 'data'
+import { actions, selectors, model } from 'data'
 
 export default () => {
```

Comment:
@schnogz Are these necessary? Is there a transaction feed style component for BSV?

Reply:
Yes